server/routes/v1/tokenDiscovery.js

```diff
@@ -3,12 +3,23 @@ const rippled = require('../../lib/rippled');
 
 const log = require('../../lib/logger')({ name: 'token discovery' });
 
-const TIME_INTERVAL = 1000 * 60 * 5; // 5 minutes
+// whether this is running in the prod environment (or in dev/staging)
+// for the purpose of running locally, this equals false if the env var doesn't exist
+// DO NOT SET TO TRUE UNLESS YOU'RE SURE ABOUT BIGQUERY USAGE
+// aka don't let the site run after you're done using it, because it'll cost $$
+const IS_PROD_ENV = process.env.REACT_APP_MAINNET_LINK
+  ? process.env.REACT_APP_MAINNET_LINK.includes('xrpl.org')
+  : false;
+// how long the auto-caching should run in dev and staging environments
+// We want to turn it off after some time so it doesn't run when we don't need it, which costs us
+// money per BigQuery query
+const TIME_TO_TEST = 1000 * 60 * 60 * 1; // 1 hour
```

Comment:
We should consider making this like 8 hours or something, since it will still show the tables, it just wont update as often

Reply:
This is how long the updating happens - so in this case it updates every half hour, but only for an hour (then it refreshes when someone calls the endpoint again).

Reply:
IMO we're not going to need to work with that page too much now that it's been built, and it's fine if the endpoint needs to be re-refreshed a couple hours after it's been deployed in dev or staging.